include/seqan3/search/configuration/all.hpp

```diff
@@ -97,6 +93,35 @@
  *
  * \include test/snippet/search/configuration_error.cpp
  *
+ * \subsection search_configuration_subsection_output 2. Output Configuration
+ *
+ * The output configuration is closely tied to the seqan3::search_result:
+ *
+ * \copydetails seqan3::search_result
+ *
+ * #### Configuring the result type
+ *
+ * As mentioned above, we can configure which information are accessible in the seqan3::search_result.
+ * For each member function there is a respective configuration element:
+ *
+ * * seqan3::search_cfg::output_query_id
+ * * seqan3::search_cfg::output_reference_id
+ * * seqan3::search_cfg::output_reference_begin_pos
+ * * seqan3::search_cfg::output_index_cursor
+ *
+ * If you specify any of the above mentioned output configuration elements, then nothing else but the selected
+ * output information is included.
+ *
+ * \include test/snippet/search/configuration_output.cpp
+ *
+ * The index cursor is an advanced data structure that lets you navigate within the index.
+ * See seqan3::fm_index_cursor and seqan3::bi_fm_index_cursor for more information.
+ * If you don't need the reference id nor the position, returning only the cursor is faster.
```

Comment:
```suggestion
 * If you don't need the reference id nor the position, returning only the cursor can be faster. This is, because the operation to get the positions can be compute intensive depending on the underlying index structure.
```

Reply:
not resolved?

Reply:
it is resolved. I added the sentence